instance/models/mixins/domain_names.py

```diff
@@ -179,6 +182,9 @@ def get_load_balanced_domains(self):
             self.internal_discovery_domain,
             self.internal_ecommerce_domain,
         ]
+        custom_domains = self.extra_custom_domains.split("\r\n")
+        print(custom_domains)
+        domain_names += [domain for domain in custom_domains if domain.endswith(self.internal_lms_domain)]
```

Comment:
Do you think it would be a good idea storing just the subdomains and concatenating the lms domain here instead?

Reply:
@viadanna We could do that, but then we should probably change how we save the rest of the subdomains, i.e. `self.internal_*_domain` and `self.external_*_domain`.
Or else some of the domains pertaining to an instance will be the complete name and some others would be the name minus the lms domain.

Reply:
Alright, makes sense. On second thought, this gives more flexibility as well.